ignite/metrics/regression/pearson_correlation.py

```diff
@@ -119,5 +119,12 @@ def compute(self) -> float:
         y_var = self._sum_of_y_squares / n - y_mean * y_mean
         y_var = torch.clamp(y_var, min=0.0)
 
-        r = cov / torch.clamp(torch.sqrt(y_pred_var * y_var), min=self.eps)
+        denom = y_pred_var * y_var
+        denom = torch.clamp(denom, min=self.eps)
+        denom = torch.sqrt(denom)
+        r = cov / denom
+
+        if torch.isnan(r):
+            return 0.0
```

Comment:
Revert this, I do not think this should be done right now. I also do not see a test updates for this metric